segbot_sensors/src/segbot_sensors/battery_monitor.cpp

```diff
@@ -0,0 +1,127 @@
+#include "ros/ros.h"
+#include "ros/time.h"
+#include <vector>
+#include "std_msgs/Header.h"
+#include "std_msgs/Float32.h"
+#include "diagnostic_msgs/DiagnosticStatus.h"
+#include "diagnostic_msgs/DiagnosticArray.h"
+#include "diagnostic_msgs/KeyValue.h"
+#include "smart_battery_msgs/SmartBatteryStatus.h" //file found
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+float voltage;
+bool sentMail = false;
+void voltageCallback(const smart_battery_msgs::SmartBatteryStatus::ConstPtr& msg)
+{
```

Comment:
I've written most of the code using the following convention for opening/closing brackets.
```
for () {
  // stuff
}

if () {
  // stuff
} else {
  // stuff
}